protocol/shannon/context.go

```diff
@@ -98,6 +102,34 @@ func (rc *requestContext) HandleServiceRequest(payload protocol.Payload) (protoc
 	return relayResponse, nil
 }
 
+// HandleWebsocketRequest opens a persistent websocket connection to the selected endpoint.
+// Satisfies the gateway.ProtocolRequestContext interface.
+// TODO_HACK(@commoddity, WebSockets): Utilize this method once the Shannon protocol supports websocket connections.
+func (rc *requestContext) HandleWebsocketRequest(req *http.Request, w http.ResponseWriter, logger polylog.Logger) error {
+	var selectedEndpointURL string
+	if rc.selectedEndpoint != nil {
+		selectedEndpointURL = rc.selectedEndpoint.PublicURL()
+	}
+
+	var upgrader = websocket.Upgrader{CheckOrigin: func(r *http.Request) bool { return true }}
+	clientConn, err := upgrader.Upgrade(w, req, nil)
+	if err != nil {
+		logger.Error().Err(err).Msg("Error upgrading websocket connection request")
```

Comment:
Do you mind adding a bit more details so tha it's easy for us to identify where the error happened when we're debugging? E.g. service_id, app_id, supplier_id?

Something for humans.